samples/snippets/batch_write_spans.py

```diff
@@ -29,7 +29,7 @@ def batch_write_spans(project_id: str):
     trace_id = str(uuid.uuid4()).replace("-", "")
     span_id = str(uuid.uuid4()).replace("-", "")[:16]
 
-    end_time = datetime.datetime.now()
+    end_time = datetime.datetime.now(tz=datetime.timezone.utc)
```

Comment:
This is fine. You could also have replaced the old datetime.now() with datetime.utcnow().